cmd/transcriber/call/transcriber.go

```diff
@@ -31,6 +34,10 @@ type Transcriber struct {
 	liveTracksWg sync.WaitGroup
 	trackCtxs    chan trackContext
 	startTime    atomic.Pointer[time.Time]
+
+	transcriberQueueCh chan captionPackage
+	transcriberWg      sync.WaitGroup
+	transcriberDoneCh  chan struct{}
```

Comment:
Is there a reason we couldn't use the existing global `doneCh`?

Reply:
Yes, the captionsPool needs to be closed, then transcriptions happen, then global `doneCh` is called. I could rearrange that, but then logically it's not as clean.